frontend/agent.py

```diff
@@ -0,0 +1,117 @@
+import json
+import logging.config
+import os
+import signal
+import sys
+from pathlib import Path
+
+from dotenv import load_dotenv
+from elevenlabs.client import ElevenLabs
+from elevenlabs.conversational_ai.conversation import Conversation, ConversationInitiationData
+from elevenlabs.conversational_ai.default_audio_interface import DefaultAudioInterface
+
+load_dotenv()
+
+agent_id = os.getenv("AGENT_ID")
+api_key = os.getenv("ELEVENLABS_API_KEY")
+
+logger = logging.getLogger("hospital_logger")
+config_file = Path("./logger_config.json")
+with open(config_file) as f:
+    config = json.load(f)
+logging.config.dictConfig(config)
+
+if not api_key:
+    logger.error("Error: ELEVENLABS_API_KEY environment variable is not set")
+    sys.exit(1)
+
+if not agent_id:
+    logger.error("Error: AGENT_ID environment variable is not set")
+    sys.exit(1)
+
+client = ElevenLabs(api_key=api_key)
```

Comment:
(Optional) You can improve passing inputs to the app using:
https://docs.pydantic.dev/latest/concepts/pydantic_settings/

Reply:
@allinorcannon What do you think? Do we have time for this?